epub33/rs/index.html

```diff
@@ -311,8 +350,7 @@ <h4>Metadata</h4>
 					<dt id="sec-opf-dclanguage">The <code>language</code> element</dt>
 					<dd>
 						<p>The language(s) of the <a>EPUB Publication</a> specified in <code>language</code> elements
-							are advisory. Reading Systems MUST use the language information associated with a resource
-							to determine its language.</p>
+							are advisory.</p>
```

Comment:
The point is: it _is_ advisory indeed. Put it another way, it is metadata about the publication proper, but there is no processing requirement added to it.

The only thing it may influence is the user interface behaviour of the reading system, of the book catalogues, etc. It does _not_ control the language of the individual textual items in the metadata (that is what `xml:lang` is for) or the language of the individual content document (which have their own way of setting the language, encoded in the respective standards.

I made some adjustments, though, here is how it looks now:

> The language(s) of the <a>EPUB Publication</a> specified in <code>language</code> elements are informational. This information MAY be exposed to the user through the Reading System specific user interface.
	
Is that better?

Reply:
Yes, much clearer, thanks. (Sorry, it's hard to ask good directional questions sometimes because I only half-understand what these attributes do :)